packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+| **Log per client**                          | Rate limiting is still based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values, but client-specific rate limiting violation information is logged by firing System Log events. |
+| **No action**                                | Rate limits aren't enforced at the client-specific level. Rate limiting is based on the [org-wide Rate Limit](/docs/reference/rate-limits/) values. |
+
+### System Log events
+
+When an individual user violates the per minute requests limit assigned to them, depending on which mode is set, the following System Log event is fired:
+
+* `system.client.rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.violation`: This event is fired when the framework is in **Enforce and log per client** mode. This event is fired when a specific client, IP address, or device identifier combination makes more than two concurrent requests. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.rate_limit.notification `: This event is fired when the framework is in **Log per client** mode. This event is fired when a specific client, IP address, or device identifier combination exceeds the total limit of 60 requests per minute. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, device identifier, and the actual user if the user already has a valid session.
+
+* `system.client.concurrency_rate_limit.notification`: This event is fired when the framework is turned on in **Log per client** mode. This event is fired when a specific client, IP address, Device token combination makes more than two concurrent requests. However, the end user wouldn't see a rate-limit violation. The System Log contains information about the client ID, IP address, Device identifier, and the actual user if the user already has a valid session.
+
+### Check your rate limits with Okta Rate Limit headers
+
+The Rate Limit headers returned when client-based rate limiting is enabled are very similar to the headers returned today through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide Rate Limit values. Okta provides three headers in each response to report client-specific rate limits.
```

Comment:
..the headers returned through the.. (no need for "today")

org-wide **R**ate **L**imits (consistent capitalization of Rate Limits - I assume this is a proper noun so needs caps throughout?)